tools/mksurfdata_esmf/gen_mksurfdata_namelist.xml

```diff
@@ -0,0 +1,317 @@
+<?xml version="1.0"?>
+
+<?xml-stylesheet type="text/xsl" href="namelist_defaults.xsl"?>
+
+<rawdata_files>
+
+  <!-- ======================== -->
+  <!-- lai -->
+  <!-- ======================== -->
+
+  <mksrf_flai>
+    <entry>
+      <data_filename>lnd/clm2/rawdata/pftcftlandusedynharv.0.25x0.25.MODIS.simyr1850-2015.c170412/mksrf_lai_78pfts_simyr2005.cdf5.c220206.nc</data_filename>
```

Comment:
- [ ] We need to update this to the new LAI file that @lawrencepj1 created

Reply:
@lawrencepj1 to confirm, should this be updated to
`pftcftdynharv.0.05x0.05.LUH2.histsimyr2005.c190116/mksrf_lai_histclm52deg005_earthstatmirca_2005.cdf5.c220228.nc`
or a different file?

Reply:
Thanks @ekluzek and @slevisconsulting yes this is the correct file for updated monthly lai, sai, and canopy height top and bottom values

Reply:
...same clarification needed for this file as for the soil color file. Thank you, @lawrencepj1 and @ekluzek